src/test/java/fiori/screens/CommonDialog.java

```diff
@@ -26,4 +31,16 @@ public int getRowsNumberFromTable(String dialogTitle) {
     public void clickHyperlinkByNameInTableColumn(String dialogTitle, String hyperlinkValue, String columnName) {
         getDialogByTitle(dialogTitle).getTable().getColumn(columnName).getRow(hyperlinkValue).getCell(columnName).getHyperlink().click();
     }
+
+    public void selectRowWithCheckedCheckbox(String columnHeader, String dialogTitle) {
+        List<WebElement> rows = getDialogByTitle(dialogTitle).getTable().getRows();
+        functionPassed(() -> {
+            for (WebElement row : rows) {
+                if (getDialogByTitle(dialogTitle).getTable().getRow(rows.indexOf(row)).getCell(columnHeader).getCheckbox().isChecked()) {
+                    getDialogByTitle(dialogTitle).getTable().getRow(rows.indexOf(row)).getCell(columnHeader).getHyperlinkByCheckbox().click();
```

Comment:
I guess code should be implementation in component classes. For example:
getDialogByTitle(dialogTitle).getTable().getRowWithCheckedCheckbox(column).select()